src/ROUTES.js

```diff
@@ -103,7 +103,7 @@ export default {
     getReportDetailsRoute: reportID => `r/${reportID}/details`,
     REPORT_SETTINGS: 'r/:reportID/settings',
     getReportSettingsRoute: reportID => `r/${reportID}/settings`,
-    TRANSITION_FROM_OLD_DOT: 'transition',
+    TRANSITION: 'transition',
```

Comment:
I think this was renamed because this route is used for two things now:
1. OldDot --> NewDot
2. NewDot (web) --> NewDot (desktop)

so the name "transition from olddot" didn't make sense anymore. I think `TRANSITION` is a bit ambiguous and could mean many different things which doesn't make it very intuitive.

How about calling it `TRANSITION_BETWEEN_APPS`?

Reply:
@marcaaron Not sure if you saw this or not, but you commented about the same thing and this was my reasoning for the suggestion.

Reply:
Thanks, maybe you can help me understand this because I'm really struggling to see it.

How does this route allow us to go from NewDot web to NewDot desktop?

And if we are using it for that purpose - maybe we should separate these concerns a bit and using some more specific "components" or "routes" that have a more specific purpose?

Reply:
It happens in this:

```
        if (Str.startsWith(window.location.pathname, Str.normalizeUrl(ROUTES.TRANSITION_BETWEEN_APPS))) {
            App.beginDeepLinkRedirectAfterTransition();
            return;
        }
```

Then `App.beginDeepLinkRedirectAfterTransition();` opens up desktop with a deeplink. My understanding is that there are places where OldDot deeplinks directly to the Desktop app (it seems like this might have started in https://github.com/Expensify/App/issues/16083 or earlier), and the only possible way for that to happen is to use NewDot web as a middle-man.

There is a fairly long discussion about this in the issue: https://github.com/Expensify/App/issues/17391#issuecomment-1515999087

Reply:
> maybe we should separate these concerns a bit and using some more specific "components" or "routes" that have a more specific purpose?

I believe this was also mentioned in the discussion a little bit, but there wasn't a strong push for it. I honestly wouldn't mind separating them out either, but it could require changing some things on OldDot (not too big a deal though probably).